cmd/internal/cli/delete.go

```diff
@@ -97,6 +97,14 @@ var deleteImageCmd = &cobra.Command{
 			sylog.Fatalf("Error parsing library ref: %v", err)
 		}
 
+		var libraryURI string
+		if deleteLibraryURI != "" {
+			libraryURI = deleteLibraryURI
+		} else if imageRef.Host != "" {
+			// override libraryURI if ref contains host name
+			libraryURI = "https://" + imageRef.Host
```

Comment:
I'm not sure what should happen if someone supplies `--library` as well as `library://host/blah`. I'm tempted to say we might want to treat this as an error? If the purpose of the `imageRef` is to (possibly) specify its location, it's confusing if it _doesn't_ delete from the right location because `--library` is set?

Reply:
Yeah - this is a silly oversight on my part as it's already handled that way for `pull`

Reply:
Please see amended code.